static/src/javascripts/projects/commercial/modules/hosted/video.js

```diff
@@ -73,6 +78,22 @@ define([
 
     function init() {
         return new Promise(function (resolve) {
+            var $youtubeIframe = $('.js-hosted-youtube-video');
+            $youtubeIframe.each(function(el){
```

Comment:
We have hosted articles (okay it's [just one](https://www.theguardian.com/advertiser-content/chester-zoo-act-for-wildlife/ensuring-a-future-for-south-asian-wildlife)) with more than one inline video, so I wanted to future-proof it in case we want youtube videos there too.